src/FortuneLevel.java

```diff
@@ -0,0 +1,4 @@
+public enum FortuneLevel {
+    大吉, 吉, 凶;
```

Comment:
Enumは英語で定義しましょう。
```suggestion
    DAIKICHI, KICHI, KYO;
```
または
```suggestion
    EXCELLENT, GOOD, BAD;
```

Reply:
EXCELLENT（"大吉"）のようにして、書き換えました。

本当はおみくじ結果に"大吉"を表示させたいのですが、うまくいかず。
getterを作成して、dispResultで取得する方法、mainで取得する方法などいろいろ試したのですが、正解にたどり着きませんでした。アドバイスいただけないでしょうか？